### ISSUE TITLE
Quickly open entry search by pulling/swiping down

### ISSUE THREAD

**Issue by SamweisGamgee** [Kunzisoft/KeePassDX#1056]
**Is your feature request related to a problem? Please describe.**
When I want to open the search I need to click on the icon on the top right corner on my phone (which is not that small and not that easy to reach).

**Describe the solution you'd like**
The solution I'd like would be a function for quickly opening the search by a gesture by doing a swipe downwards. This of course should only work if you already are on top of the entry list so that the swipe/pull down does not come into conflict with the normal scrolling of the entries.

**Describe alternatives you've considered**
Alternatively there could be a function to move the search button down to the bottom of the device to the "add new entry" for example, so that it's easier to reach with the thumb.

**Comment by vertigo220** [comment]
Definitely agree, as phones have become so large, I hate having to reach way up to the top to access functions. Even the sort button would be nice to be accessible from the bottom, but the search button is especially important for this. I'm not sure about the swiping down, though, since I'd like to be able to quickly access it from anywhere. Another option would be to put it in the side menu, but IME side swiping on Android is pretty abysmal, and it's about a 50/50 chance, or worse, that when I try to open the menu it will actually open vs swiping one of the items on the screen or even registering as an up/down swipe.

At the very least, an option to simply move or even duplicate the search button to the bottom would be nice, as well as possibly the sort button, or there could be an option to simply move the entire title bar to the bottom. Another option, to prevent to much clutter, would be a single button in place of the + that, when clicked, expands to multiple buttons or shows a pop-up box with a list of actions.

**Comment by dstrelnikov** [comment]
Yes, I think this feature would be a great balance between opening or not opening the search on database unlock (which are the current options). Once the database is unlocked, it is very convenient to swipe down to open the search.
